scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -449,34 +469,42 @@
   (getSoftConstraints [_] [])
   (taskGroupName [_] (str (:job/uuid job))))
 
+(def adjust-job-resources-for-pool-fn
```

Comment:
Even though this isn't a `defn`, I still think it's weird to suffix the name with `-fn` for top-level definitions.